test/test_mifare_desfire_key.c

```diff
@@ -65,7 +74,7 @@ test_mifare_desfire_key(void)
     cut_assert_equal_int(0x02, version, cut_message("Wrong MifareDESFireKey version"));
     mifare_desfire_key_free(key);
 
-    uint8_t key3_3des_data[16] = { 0x00, 0x00, 0x00, 0x01, 0x00, 0x00, 0X00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x77 };
+    uint8_t key3_3des_data[16] = { 0x00, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x77 };
```

Comment:
:+1: for minor style fix (there is at least another one at the same position on line 65 :wink:

Reply:
fixed.